src/qaqc/remove_flag_method.R

```diff
@@ -133,15 +127,14 @@ verify_flag_data <- function(weekly_plot_object, daily_plot_object) {
 # Generating the list of plots (this example only has one plot that has a repeated value flag)
 # we are doing it this way so that we can get the data from the daily ggplot
 test_weekly <- generate_weekly_flag_plots("archery", "Temperature", "slope violation")
```

Comment:
My selecting a specific flag, could we be verifying the same data multiple times and redoing the work again? I wonder if the function above can also remove any plots which already have verification == TRUE

Reply:
This is true! 

To circumvent re-reviewing data the weekly and daily plot generation functions have the ability to print all the plots with flags if the flag_arg is NULL. This makes it so that if any flag is found in a date, it will just print that plot without the need to specify a flag. Since I am testing I wanted to print out some of the flags and not all of the flags for speed.

I do think that we will still need to make a simple system that tracks which data has been reviewed and which data has not been reviewed, though. A column that has verification == TRUE, or something along those lines is a good idea.